go.mod

```diff
@@ -99,7 +99,7 @@ require (
 	golang.org/x/time v0.3.0
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231016165738-49dd2c1f3d0b
 	google.golang.org/grpc v1.59.0
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/protobuf v1.33.0
```

Comment:
I think similar to https://github.com/moby/buildkit/pull/4318#pullrequestreview-1669389325 we should have this done in containerd repo first

Reply:
Yeah, containerd should also be updated; that said, I'm not sure if that should always be the rule; it's a direct dependency here. We had a quick chat about this in the Moby maintainers call last Thursday (as we updated this dependency in Moby); consensus there was that we had enough coverage in the moby repository to validate it worked, so we went ahead and merged the update.